Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -464,10 +557,55 @@ ViewHelpers with the `useNonce` attribute:
         .some-style { color: red; }
     </f:asset.css>
 
+.. _content-security-policy-nonce-caching:
+
+Notes about nonces and caching
+------------------------------
+
+Nonces are implemented via a PSR middleware and thus applied dynamically. This
```

Comment:
I get that caching is a problem, but do you offer a solution?

Reply:
Yes, the section has a solution!

Reply:
Ack, I know not about the topic